### ISSUE TITLE
Doesn't work with go modules [go v.1.11.2] 

### ISSUE THREAD

**Issue by Terminator637** [devimteam/microgen#69]
In a project that uses go modules, I run the microgen command and get the following:
```
@microgen 0.9.1
fatal: not in GOPATH
```

**Comment by vetcher** [comment]
yes, I know about this problem with go modules. Microgen uses GOPATH to find import path to package with interface. I want to release v1 in near future which will support go modules.

**Comment by Terminator637** [comment]
> yes, I know about this problem with go modules. Microgen uses GOPATH to find import path to package with interface. I want to release v1 in near future which will support go modules.

Thank you! Will wait for new release!

**Comment by nicocesar** [comment]
I know is 2020, but any news on this?

**Comment by vetcher** [comment]
Hi. For this years I realise, that core idea of microgen (generate everything from interface) is wrong. And, what's more important, I do not have access rights to this repo, because I left devimteam 2 years ago.

But! We have benefits of open source: you may fork it and continue.
I do not recommend use `master` branch, because its really out of date. You may try branch `v22` (it's that branch with refactoring, that should work with modules; do not look at readme) https://github.com/devimteam/microgen/tree/v22.
As I remember, 2 years ago it works fine.

Also, you may look at https://github.com/hexdigest/gowrap
